### PR TITLE
7583 quick update pricing page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7583

### CODE REVIEW

File: source/partials/_pricing-faq-university.erb [hunk 1]
```diff
@@ -120,6 +120,24 @@
...
+          </a>
+          <i class="indicator glyphicon-toggle glyphicon-toggle-plus pull-right"></i>
+        </h4>
+      </div>
+      <div id="un-boss" class="panel-collapse collapse">
+        <div class="panel-body">
+          We know there are lots of people that you might need to involve in this decision. And we
+          know jumping through all the bureaucratic hurdles can be a pain. So, we're here to help.
+          We are available to demo to yourself and a group of your colleagues. Or alternatively, you
+          can <a href="/img/assets/pricing/Codio_Brochure_Print.pdf">download our brochure</a> that describes all the Codio product features.
```
Comment (destitutus) [inline source/partials/_pricing-faq-university.erb]: did you check - it start downloading or open pdf in current window?

Comment (denpira) [inline source/partials/_pricing-faq-university.erb]: Check in ff and chrome - open pdf in current window

Comment (destitutus) [inline source/partials/_pricing-faq-university.erb]: maybe make sense open it in new window? cc @MaximKraev

Comment (MaximKraev) [inline source/partials/_pricing-faq-university.erb]: please add target new tab if it doesn't stay open after download started

Comment (denpira) [inline source/partials/_pricing-faq-university.erb]: Done

**Comment by destitutus** [comment]
👍

**Pull request merged**